app/views/additional_details/questionnaires/_form.html.haml

```diff
@@ -7,9 +7,10 @@
       - if questionnaire.errors.any?
         .alert.alert-danger
           %ul.list-unstyled
-            - questionnaire.errors.full_messages.each do |msg|
-              %li
-                = msg
+            - questionnaire.errors.messages.values.each do |msg|
```

Comment:
You can, but you'll end up with 'Item option content cant be blank', instead of "Content can't be blank" which is what they wanted.